Makefile

```diff
@@ -4,11 +4,11 @@ LDFLAGS ?=
 TMPDIR ?= /tmp
 
 VCFILE := v.c
-TMPVC  := $(TMPDIR)/vc
-TMPTCC := /var/tmp/tcc
+VC  := ./vc
+TCC := ./tcc
```

Comment:
Unfortunately, that can not work, TCC should be installed in a specific hardcoded folder, not in an arbitrary one, since it contains the name inside itself.

Reply:
The binary version from https://github.com/vlang/tccbin is configured with:
`./configure --prefix=/var/tmp/tcc --crtprefix=/var/tmp/tcc/lib/:/usr/lib64/:/usr/lib/x86_64-linux-gnu/`

Reply:
VC does not have that restriction, so it can be installed in a local vc/ folder.